spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -70,179 +70,154 @@
-        rules_and_groups = selected.map { |r| r.attributes['idref'].value }
-        rules_only = rules_and_groups.select { |r| r.exclude?('content_rule_group') }
-
-        expect(response).to have_http_status :ok
-        expect(response.headers['Content-Type']).to eq('application/xml')
-        expect(tailoring_values).to eq(tailored_values.map(&:value))
-        expect(rules_only).to match_array(item.rules_added.map(&:ref_id))
-        expect(rules_and_groups - rules_only).to match_array(item.rule_group_ref_ids)
-      end
-    end
+      context 'with unauthorized policy' do
+        let(:extra_params) do
+          {
+            # policy of a foreign account
+            policy_id: FactoryBot.create(:v2_policy, account: FactoryBot.create(:v2_account)),
+            id: item.id
+          }
+        end
 
-    context 'with unauthorized policy' do
-      let(:canonical_profile) { FactoryBot.create(:v2_profile) }
+        let(:item) { FactoryBot.create(:v2_tailoring, policy: parent, os_minor_version: 8) }
 
-      let(:extra_params) do
-        {
-          id: item.id,
-          policy_id: FactoryBot.create(:v2_policy, account: FactoryBot.create(:v2_account))
-        }
-      end
+        it 'results in 404 error' do
+          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
 
-      let(:item) do
-        FactoryBot.create(:v2_tailoring, policy: parent, profile: canonical_profile, os_minor_version: 8)
+          expect(response).to have_http_status :not_found
+        end
       end
 
-      it 'results in 404 error' do
-        get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
+      context 'with mismatching set of IDs' do
+        let(:extra_params) do
+          {
+            policy_id: FactoryBot.create(:v2_policy, account: FactoryBot.create(:v2_account)),
+            id: Faker::Internet.uuid
+          }
+        end
+
+        it 'results in 404 error' do
+          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
 
-        expect(response).to have_http_status :not_found
+          expect(response).to have_http_status :not_found
+        end
       end
     end
 
-    context 'with mismatching set of IDs' do
-      let(:canonical_profile) { FactoryBot.create(:v2_profile) }
-
-      let(:extra_params) do
-        {
-          id: Faker::Internet.uuid,
-          policy_id: FactoryBot.create(:v2_policy, account: current_user.account)
-        }
+    context 'authorized via cert_auth' do
+      before do
+        allow(controller).to receive(:rbac_allowed?).and_call_original
+        allow(controller).to receive(:any_inventory_hosts?).and_return(true)
       end
 
-      it 'results in 404 error' do
-        get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
+      let(:account) { FactoryBot.create(:v2_account, :cert_authorized) }
```

Comment:
Do we need the account to be defined as a separate `let`? Can't we merge it into the block beneath?